docs/EN_US/ECLStandardLibraryReference/SLR-Mods/CurrentTime.xml

```diff
@@ -39,8 +39,11 @@
 
   <para>The <emphasis role="bold">CurrentTime</emphasis> function returns the
   current time. If the in_local_time parameter is TRUE the returned value is
-  local to the cluster computing the time, if FALSE then the UTC is
-  returned.</para>
+  local to the cluster computing the time, if FALSE then the UTC is returned.
+  </para>
```

Comment:
question: @JamesDeFabia why is there this whitespace change?  Is it something introduced by whatever edits the file?